R/zzz.R

```diff
@@ -1,6 +1,10 @@
 .lockbox_env <- new.env(parent = emptyenv())
 
 set_session_id <- function() {
+  if (isTRUE(getOption("lockbox.disable_sessions"))) {
+    return()
+  }
```

Comment:
I've generally used `return()` for functions that don't expect to return values anyway and are only called for their side effects.